### PR TITLE
Pull Request : Adding java doc in Component Report Component

### PR DESCRIPTION
**Pull request by Shivamraghu1208**

### CODE REVIEW

File: core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java [hunk 1]
```diff
@@ -24,26 +24,57 @@
...
 public class ComponentReportModel {
 
+    /**
+     * The title - title of Component
+     */
     @ValueMapValue
     private String title;
 
+    /**
+     * The dropDownPlaceHolder - Name of dropdown
```
Comment (abhinav-agarwal) [inline core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java]: incorrect javadoc

File: core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java [hunk 2]
```diff
@@ -24,26 +24,57 @@
...
+     */
     @ValueMapValue
     private String dropDownPlaceHolder;
 
+    /**
+     * The request -  SlingHttpServletRequest object
+     */
+    @SlingObject
+    private SlingHttpServletRequest request;
```
Comment (abhinav-agarwal) [inline core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java]: do we need this object?

Comment (Shivamraghu1208) [reply core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java]: Yess

Comment (abhinav-agarwal) [reply core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java]: will your compoent  work in publish mode?

Comment (Shivamraghu1208) [reply core/src/main/java/com/adobe/aem/sample/site/core/models/ComponentReportModel.java]: sir when we are using SlingHttpServletRequest request then the component is not working on publish mode. But now i am taking resource resolver from ResourceresolverFactory  now it is working in publish mode

**Review by abhinav-agarwal — APPROVED** [review]

**Pull request merged**